Ginger/Ginger/Variables/VariableEditPages/VariableDateTimePage.xaml.cs

```diff
@@ -106,11 +107,26 @@ private void dtpInitialDate_TextChanged(object sender, EventArgs e)
             }
         }
 
-        private void txtDateFormat_LostFocus(object sender, System.Windows.RoutedEventArgs e)
+        private void UpdateIntialDateTimePicker()
         {
             dtpInitialDate.CustomFormat = txtDateFormat.Text;
             dpMinDate.CustomFormat = txtDateFormat.Text;
             dpMaxDate.CustomFormat = txtDateFormat.Text;
         }
+
+        private async void txtDateFormat_TextChanged(object sender, TextChangedEventArgs e)
+        {
+            // this inner method checks if user is still typing
+            async Task<bool> UserKeepsTyping()
+            {
+                await Task.Delay(1000);
+                UpdateIntialDateTimePicker();
+                return true;
+            }
+            if (await UserKeepsTyping())
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5706243)